lms/djangoapps/course_api/blocks/views.py

```diff
@@ -85,6 +87,15 @@ class BlocksView(DeveloperErrorViewMixin, ListAPIView):
 
           Example: return_type=dict
 
+        * block_types: (list) Requested types of blocks. Possible values include sequential,
+          vertical, html, problem, video, and discussion.
+
+          Example: block_types=vertical,html
+
+        * lti_url: (boolean) Indicates if it's required to show the field 'lti_url'.
```

Comment:
Should follow other examples with optional/default details.  This may also need to wait on the discussion about how to add this to the api above (more generically, or one field at a time).

If this stayed, I'd change language to:
Indicates whether to include the field 'lti_url'.